README.md

```diff
@@ -397,6 +397,8 @@ GRDB can be installed as a framework, or a static library.
 
 The [Swift Package Manager](https://swift.org/package-manager/) automates the distribution of Swift code. To use GRDB with SPM, add a dependency to `https://github.com/groue/GRDB.swift.git`
 
+> GRDB offers a `static` as well as `dynamic` library. Choose the default `GRDB` (static) library if you're not sure about the difference. Choose `GRDB-dynamic` if you are going to link it with multiple targets within your app and only wish to link to a shared, dynamic framework once. See [#How to link as dynamic](https://forums.swift.org/t/how-to-link-a-swift-package-as-dynamic/32062)
+>
```

Comment:
The default `GRDB` library is not static. It's the unfortunate behavior of Xcode that treats it as static only. But maybe Xcode will improve ;-) So I suggest instead:

```suggestion
GRDB two libraries, `GRDB` and `GRDB-dynamic`. Pick only one. When in doubt, prefer `GRDB`. The `GRDB-dynamic` library can reveal useful if you are going to link it with multiple targets within your app and only wish to link to a shared, dynamic framework once. See [#How to link as dynamic](https://forums.swift.org/t/how-to-link-a-swift-package-as-dynamic/32062)

```

Reply:
Yes that's correct, makes sense.

Reply:
Besides this wording change, the PR looks pretty good. I'll run the added test on my machine shortly.

I've read your conversation on https://forums.swift.org/t/specify-dependencies-should-be-built-as-dynamic-or-static/35257, and the resistance you find doesn't change my mind.

Quoting @SDGGiesbrecht there:

> If his IDE lacks a way to control the linking directly, the top‐level user should create his own package with a single .dynamic library, which in turn depends (statically) on all nodes that are common to the various executable components he plans to produce. Then each executable should depend solely on that library instead.

I can't disagree. But I also clearly see how unergonomic this is. And [I'm not the only one](https://github.com/groue/GRDB.swift/pull/1165#issuecomment-1028739487).

Your pull request is _pragmatic_, and _does not harm_. 👍